yt_dlp/extractor/tver.py

```diff
@@ -3,75 +3,40 @@
 
 
 from .common import InfoExtractor
-from ..compat import compat_str
 from ..utils import (
-    ExtractorError,
-    int_or_none,
-    remove_start,
     smuggle_url,
+    str_or_none,
     traverse_obj,
 )
 
 
 class TVerIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(?P<path>corner|episode|feature|lp|tokyo2020/video)/(?P<id>[fc]?\d+)'
+    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(?P<path>corner|episodes?|feature|lp|tokyo2020/video)/(?P<id>[a-zA-Z0-9]+)'
```

Comment:
Is tver.jp/lp/{video_id} still a valid pattern?

In the `api_response` I saw it being used like a shortcut `tver.jp/lp/episodes/{video_id}`.

Perhaps:
`r'https?://(?:www\.)?tver\.jp/(?:lp/)?(?P<path>corner|episodes?|feature|tokyo2020/video)/(?P<id>[a-zA-Z0-9]+)'`?

Reply:
I haven't seen `corner` and `feature` where the former was the majority of the four. Also `tokyo2020` is already over (as it means the Olympics Game held in 2021, Tokyo)